router-tests/testenv/testenv.go

```diff
@@ -223,6 +223,7 @@ type Config struct {
 	SubgraphAccessLogsEnabled          bool
 	SubgraphAccessLogFields            []config.CustomAttribute
 	AssertCacheMetrics                 *CacheMetricsAssertion
+	DisableSimulateCloudExporter       bool
```

Comment:
I need a bit more clarity on this, because we don't have exporters in tests and basically all of our tests kind of "simulate" to have a cloud exporter.